newsroom/agenda/agenda.py

```diff
@@ -682,6 +699,17 @@ def set_delivery(self, wire_item):
         if not wire_item.get('coverage_id'):
             return
 
+        def is_delivery_validated(coverage, item):
+            latest_delivery = get_latest_available_delivery(coverage)
+            if not latest_delivery or not item.get('rewrite_sequence'):
+                return True
+
+            if item.get('rewrite_sequence', 0) >= latest_delivery.get('sequence_no', 0) \
+                    or item.get('firstpublished') >= latest_delivery.get('publish_time'):
```

Comment:
@nrvikas can you parse dates (using https://github.com/superdesk/newsroom/blob/master/newsroom/utils.py#L77) and  then compare